src/app/pages/candidate-details/candidate-details.page.html

```diff
@@ -14,12 +14,12 @@
     <ion-grid>
         <ion-row>
             <ion-col size="80">
-                <h2>
+                <h2 id="candidates-name">
                     {{ pageState.name }}
                 </h2>
             </ion-col>
             <ion-col size="16">
-                <h2 class="test-start-time">
+                <h2 id="time" class="test-start-time">
```

Comment:
same again, "test-start-time" instead of just "time"

Reply:
Change to reflect commit